plugins/modules/sssd_info.py

```diff
@@ -0,0 +1,300 @@
+#!/usr/bin/python
+# Copyright (c) 2025 Aleksandr Gabidullin <qualittv@gmail.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import annotations
+
+
+DOCUMENTATION = r"""
+---
+module: sssd_info
+version_added: 12.2.0
+short_description: Check SSSD domain status using D-Bus
+description:
+    - Check the online status of SSSD domains, list domains, and retrieve active servers using D-Bus.
+author: "Aleksandr Gabidullin (@MikeyTide)"
+requirements:
+    - dbus-python
+attributes:
+    check_mode:
+        support: full
+    diff_mode:
+        support: none
+    platform:
+        platforms: posix
+        description: This action requires a system with D-Bus and SSSD running.
+        support: full
+options:
+    action:
+        description:
+            - The action to perform.
+            - V(domain_status) - Check if domain is online.
+            - V(domain_list) - List all configured domains.
+            - V(active_servers) - Get active servers for domain.
+            - V(list_servers) - List all servers for domain.
+        type: str
+        required: true
+        choices: ['domain_status', 'domain_list', 'active_servers', 'list_servers']
+    domain:
+        description:
+            - Domain name to check.
+            - Required for O(action=domain_status), O(action=active_servers) and O(action=list_servers).
+        type: str
+    server_type:
+        description:
+            - The type of server to retrieve for the O(action=active_servers) and O(action=list_servers).
```

Comment:
I would also add a line here making it clear to the users that this param is required for those two cases.